views/event_details.hbs

```diff
@@ -20,21 +20,27 @@
   <div class="margins">
     <h3 class="dark-text">
       <i class="fa fa-map-marker map-icon" aria-hidden="true"></i>
-      {{event.place.en.name}}
+      {{event.place.[lang].name}}
```

Comment:
Need to drop the . in front of `lang`

Reply:
Actually no you can't do this - you'd need to create a new value and add it to `result.body` in `event-details.js`